### ISSUE TITLE
Visual indication that the timeline is larger then the screen

### ISSUE THREAD

**Issue by Larspolo** [eJourn-al/eJournal#353]
# Feature request

## Description
An extra indication that there are more nodes underneath the screen. Also graded entries could maybe shrink so they dont take up as much space as the more important nodes?

## User interface
![image](https://user-images.githubusercontent.com/11822478/45645819-f6fc8980-bac1-11e8-9c71-c34042e8e104.png)
Make is more clear for the user that they are not seeing all the nodes in the journal.

**Comment by engelhamer** [comment]
I do not think it is a good idea to make further distinctions in node size (for simplicity's sake), but have something in mind for this.

**Comment by engelhamer** [comment]
See 'nog 20 resultaten' at https://nos.nl/collectie/13860/artikel/2373088-nieuwe-voorlopige-prognose-vvd-35-zetels-d66-tweede-met-24-en-bij1-toch-1-zetel?fbclid=IwAR1Q401zIpRowvwNzwiuMp_DANfWlRx4s8cYBLfp66SLKlsg2DVtnulcSS4

![image.png](https://zube.io/files/ejourn-al/9aa4a4f0044a1bf9871a9a6c9f5f7e00-image.png)
